src/app/financial-chart/multiple-data/financial-chart-multiple-data.component.ts

```diff
@@ -0,0 +1,6325 @@
+import { Component } from "@angular/core";
```

Comment:
yes, the data should go somewhere it can be used by more than one sample.  i didn't see an existing pattern for this in the samples browser, so i conformed to the architecture-free arrangement.

where would you recommend putting this data?

Reply:
I'd say a separate ts file probably works for now, which you can import into that TS file

Reply:
The bog standard way would be to create a "data service", but, yeah, I still think that would be overkill for most of the samples, truly. As someone would need to decode the sample architecture to grok what we were trying to show.

Reply:
@davidnegley you might want to set title on each data source such that the FC can show titles in tooltips instead of PriceSeries1 PriceSeries2